build-tools/mono-runtimes/mono-runtimes.projitems

```diff
@@ -15,7 +15,8 @@
       <RanLib>$(_ArmRanLib)</RanLib>
       <Strip>$(_ArmStrip)</Strip>
       <ConfigureFlags>--host=armv5-linux-androideabi $(_TargetConfigureFlags)</ConfigureFlags>
-      <OutputRuntime>libmonosgen-2.0.so</OutputRuntime>
+      <OutputRuntime>libmonosgen-2.0</OutputRuntime>
+      <OutputRuntimeExtension>so</OutputRuntimeExtension>
```

Comment:
If we're going to split out the extension like this, we should alter `%(OutputProfiler)` and `%(OutputMonoPosixHelper)` to follow suit.

I'd also be inclined to change the naming convention, but I'm not aware of a *short* name for "name without extension", and `%(OutputRuntimeWithoutExtension)` looks ugly. `%(OutputRuntimeBasename)` also doesn't quite fit the bill, as normally "basename" includes the file extension.